back/engines/commercial/admin_api/app/services/admin_api/project_copy_service.rb

```diff
@@ -2,21 +2,35 @@
 
 module AdminApi
   class ProjectCopyService < ::TemplateService
-    def import(template, folder: nil)
+    def import(template, folder: nil, local_copy: false)
       service = MultiTenancy::TenantTemplateService.new
       same_template = service.translate_and_fix_locales template
-      project_ids_before = Project.ids
-      ActiveRecord::Base.transaction do
+
+      created_objects_ids = ActiveRecord::Base.transaction do
         service.resolve_and_apply_template same_template, validate: false
       end
-      Project.where.not(id: project_ids_before).each do |project|
+
+      project = Project.find(created_objects_ids[:project].first)
+      unless local_copy
         project.update!(slug: SlugService.new.generate_slug(project, project.slug))
```

Comment:
I _think_ this is why it _is_ needed for the Project Copy Tool (not our new local copy feature):

It is possible to specify a slug value for the copied project. If you specify the same slug as an existing project (on the target tenant), then this will fail due the error:
```
PG::UniqueViolation:
  ERROR:  duplicate key value violates unique constraint "index_projects_on_slug"
  DETAIL:  Key (slug)=(Your coolest tricks to cool down the city) already exists.
```
And, indeed , the `engines/commercial/admin_api/spec/services/project_copy_service_spec.rb` fails with [this error](https://app.circleci.com/pipelines/github/CitizenLabDotCo/citizenlab/82822/workflows/5bfb2352-e82d-4168-8e58-6c85ecd404a6/jobs/189606) if we remove this slug update for the existing Project Copy tool.

So, why do I skip this line for our new feature? TL,DR; the suffix numberings are nicer, but it doesn't really matter.

If we skip it, then the first copy of `source_project` will have a slug: '<source_project_title>-copy`, the second copy would have a slug: '<source_project_title>-copy-1`, etc.

 If we _don't_ skip it, then the first copy of `source_project` will have a slug: '<source_project_title>-copy-1`, the second copy would have a slug: '<source_project_title>-copy-2`, etc.
 
 So, I prefer the numbering system for slug collisions, but it is not important, so we could just apply the line to all cases.

Reply:
> but it doesn't really matter

If so, I would go with simplicity. If we get rid of this condition, we also can get rid of the `local_copy` param.

Reply:
Hmm... but I'm not sure we want to apply `set_default_topics` ... I'll check what this does exactly. It might mess up the copying of `project_topics`.

Reply:
Yes, it will set the `projects_allowed_input_topics` to these Topics:
```
DEFAULT_CODES = %w[nature waste sustainability mobility technology economy housing public_space safety education culture health inclusion community services other]
```
I think I would rather copy the exact `project_allowed_input_topics` from the source project, and then let the Admin add / remove `allowed_project_input_topics` as desired.

Reply:
> I think I would rather copy the exact project_allowed_input_topics

Indeed, makes sense.